From a76de3efbf084e981c2dfd4596458f70dc542219 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Julian=20R=C3=BCth?= <julian.rueth@fsfe.org>
Date: Thu, 16 Feb 2023 14:55:49 +0100
Subject: [PATCH] Use gp from environment

on some architectures, conda does not correctly rewrite the GP path
leading to this error when invoking new_data:

/home/conda/feedstock_root/build_artifacts/sympow_1676503161298/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/sympow/new_data: line 37: /home/conda/feedstock_root/build_artifacts/sympow_1676503161298/_build_env/bin/gp: No such file or directory
---
 new_data | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/new_data b/new_data
index 70b8d9e..6337973 100755
--- a/new_data
+++ b/new_data
@@ -26,7 +26,7 @@ SYMPOW="$SYMPOW_INVOCATIONNAME ${SYMPOW_OPTS_VERBOSITY:--quiet}"
 echo "Running the new_data script for $3"
 
 SH=$1
-GP=$2
+GP=gp
 
 echo "Making the datafiles for $3"
 echo ""
-- 
2.39.1

